Answers.md

```diff
@@ -0,0 +1,18 @@
+1. What is the difference between an adaptive website and a fully responsive website?
+    adaptive web design is different from responsive design in that there isn't one layout
+    that alwawys changes.
+
+2. Describe what it means to be mobile first vs desktop first.
```

Comment:
Also, mobile first will focus on the mobile user first and design the desktop as a modification of that. It's the opposite for desktop first